src/main/java/com/puppycrawl/tools/checkstyle/filters/JavaPatchFilterElement.java

```diff
@@ -184,9 +185,20 @@ public boolean accept(TreeWalkerAuditEvent event) {
      * @return true if it is matching
      */
     private boolean isFileNameMatching(TreeWalkerAuditEvent event) {
-        return event.getFileName() != null
-                && ((event.getFileName()).equals(fileName)
-                || event.getFileName().contains(fileName));
+        boolean result = event.getFileName() != null;
+
+        if (result) {
+            String eventFileName = event.getFileName();
```

Comment:
I'd have that before 188 to avoid the duplicate method call.

Reply:
Done.